src/overlays/actors/ovl_En_Bigpo/z_en_bigpo.c

```diff
@@ -688,7 +688,7 @@ void EnBigpo_BurnAwayDeath(EnBigpo* this, PlayState* play) {
     if (this->idleTimer < 8) {
         camYaw = Camera_GetCamDirYaw(GET_ACTIVE_CAM(play)) + 0x4800;
         if (this->idleTimer < 5) {
-            unkTemp = (this->idleTimer << 0xC) - 0x4000;
+            unkTemp = (this->idleTimer * 0x1000) - 0x4000;
             // 1.4.0...1 is NOT 1.4, the rodata demands it
             tempVec.y = (((Math_SinS(unkTemp) * 23.0f) + 40.0f) * 1.4000001f) + this->actor.world.pos.y;
```

Comment:
```suggestion
            tempVec.y = (((Math_SinS(unkTemp) * 23.0f) + 40.0f) * (1400.0f * 0.001f)) + this->actor.world.pos.y;
```
works, same for the others below. (Should also be able to remove the `f32` explicit cast in the third one)